runtime/server/timeseries_api.go

```diff
@@ -454,11 +456,23 @@ func convertRowsToTimeSeriesValues(rows *drivers.Result, rowLength int) ([]*api.
 		delete(row, "ts")
 		value.Records = make(map[string]float64, len(row))
 		for k, v := range row {
-			value.Records[k] = v.(float64)
+			switch x := v.(type) {
+			case int32:
+				value.Records[k] = float64(x)
+			case int64:
+				value.Records[k] = float64(x)
+			case float32:
+				value.Records[k] = float64(x)
+			case float64:
+				value.Records[k] = x
+			default:
+				if converr == nil {
+					converr = fmt.Errorf("unknown type %T ", v)
```

Comment:
Return the error immediately here. Returning both a valid result and an error at the end of the function is not conventional

Reply:
Should we return a partial result? Assuming we have several dozen columns and 1 is unsupported, should we present almost all the table and mark the invalid to the user or have the user wait until we release an improvement?

Reply:
I'd say the options are to either return a partial (best effort) result, or to return an error message. What situation could make the error happen – and what's the current behavior in Node?